examples/bridge-app/bridge-common/gen/IMClusterCommandHandler.cpp

```diff
@@ -34,6 +34,18 @@
 namespace chip {
 namespace app {
 
+namespace {
+void ReportCommandUnsupported(Command * aCommandObj, EndpointId aEndpointId, ClusterId aClusterId, CommandId aCommandId)
```

Comment:
could we extract it even further in the library itself? it seems to me we would want generated code to depend on dynamic bits only. General methods that do not depend on dynamic input can be placed in standalone library code that is not auto-generated.

Reply:
We could, but I expect this specific thing to change a lot based on some of the discussions @mrjerryjohns and I have had.  In particular, I expect it will become a method on Command (or some equivalent thereof)...

I could put it in some temporary place in the library and then change it later, but I'd rather not do that work twice, honestly.

Reply:
if you are confident this changes, I am ok with that answer.

My concern was 'generated code' to not really be generated but a "normal" method instead. Could add a TODO but marking conversation resolved for now.